components/ConjugationPage/ConjugationPage.js

```diff
@@ -203,3 +200,12 @@ const StyledButtonContainer = styled.div`
     color: white;
```

Comment:
ich weiß ich hatte das schon an anderer Stelle mal gesagt, aber vielleicht willst du ja die Farben mal in Variablen legen. Dann kannst du sie global verwalten :)

Reply:
das ist die nächste US :) danke für den Hinweis!